src/main/java/dev/pgm/community/polls/commands/PollVoteCommands.java

```diff
@@ -0,0 +1,27 @@
+package dev.pgm.community.polls.commands;
+
+import dev.pgm.community.Community;
+import dev.pgm.community.CommunityCommand;
+import dev.pgm.community.polls.feature.PollFeature;
+import dev.pgm.community.utils.CommandAudience;
+import org.bukkit.entity.Player;
+import tc.oc.pgm.lib.cloud.commandframework.annotations.CommandMethod;
+
+public class PollVoteCommands extends CommunityCommand {
+
+  private final PollFeature polls;
+
+  public PollVoteCommands() {
+    this.polls = Community.get().getFeatures().getPolls();
+  }
+
+  @CommandMethod("yes")
```

Comment:
I think these should be prefixed with something, maybe taking inspiration from the vote book.
`votenext -o [option]` so maybe a `votepoll -o [option]`

The commands for these actions are not expected to be typed by players but from clickable text so there's no need to make them so generic which could cause other issues.

Reply:
note, -o is for "open", not for the option, so just `/votepoll [option]`

Reply:
Great suggestion 👍 Will help pave the way for the eventual multi-answer polls.

Reply:
> note, -o is for "open", not for the option, so just `/votepoll [option]`

🤦 I added the `-o` option 😆 it was 3 years ago so please forgive me.